cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/JsonMenu/JsonLiveViewer/index.tsx

```diff
@@ -15,42 +15,28 @@
  */
 
 import Button from '@material-ui/core/Button';
-import Drawer from '@material-ui/core/Drawer';
-import List from '@material-ui/core/List';
 import withStyles, { StyleRules, WithStyles } from '@material-ui/core/styles/withStyles';
 import Tooltip from '@material-ui/core/Tooltip';
 import FullscreenExitIcon from '@material-ui/icons/FullscreenExit';
-import SaveAltIcon from '@material-ui/icons/SaveAlt';
-import {
-  downloadPluginJSON,
-  getJSONConfig,
-} from 'components/PluginJSONCreator/Create/Content/JsonMenu/utilities';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import { JSONStatusMessage } from 'components/PluginJSONCreator/Create/Content/JsonMenu';
+import PluginJSONImporter from 'components/PluginJSONCreator/Create/Content/JsonMenu/PluginJsonImporter';
 import { ICreateContext } from 'components/PluginJSONCreator/CreateContextConnect';
 import * as React from 'react';
 
-const JSON_VIEWER_WIDTH = '600px';
-
 const styles = (theme): StyleRules => {
   return {
-    jsonViewer: {
-      zIndex: theme.zIndex.drawer,
-      width: JSON_VIEWER_WIDTH,
-    },
-    jsonViewerPaper: {
-      width: JSON_VIEWER_WIDTH,
-      backgroundColor: theme.palette.white[50],
-    },
-    toolbar: {
-      minHeight: '48px',
-    },
-    mainMenu: {
-      borderTop: `1px solid ${theme.palette.grey['500']}`,
-      paddingTop: theme.Spacing(1),
-      paddingBottom: theme.Spacing(1),
-    },
     jsonActionButtons: {
-      padding: '5px',
+      padding: '0px',
       display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    currentFilename: {
+      position: 'relative',
+      margin: '0 auto',
+      left: '25px',
+      fontFamily: 'Courier New',
```

Comment:
don't use new font. for block letter, you can use `<pre>`

Reply:
Resolved